mscore/events.cpp

```diff
@@ -39,25 +39,18 @@ namespace Ms {
 
 bool ScoreView::event(QEvent* event)
       {
-      if (event->type() == QEvent::KeyPress && editData.element) {
+      if (event->type() == QEvent::KeyPress && editMode()) {
             QKeyEvent* ke = static_cast<QKeyEvent*>(event);
             if (ke->key() == Qt::Key_Tab || ke->key() == Qt::Key_Backtab) {
                   if (editData.element->isTextBase())
                         return true;
-                  bool rv = true;
-                  if (ke->key() == Qt::Key_Tab) {
-                        rv = editData.element->nextGrip(editData);
```

Comment:
next/prevGrip returns bool. Why did you decide to ignore the returned value?

Reply:
It is potentially useful to be able to know whether or not the selected grip has wrapped around to begin a new cycle, but I could not think of a way to make use of this information. Therefore, I ignored the return value, rather than capturing it to a variable that serves no purpose.